### PR TITLE
seperate LaTeXSVG into latexsvgfile.jl 

### PR DESCRIPTION
**Pull request by ArbitRandomUser**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [ ] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [ ] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [ ] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [ ] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [ ] Did I properly add test dependencies to the `test` directory (if applicable)?
- [ ] Did I check relevant tutorials that may be affected by changes in this PR?
- [ ] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**
Closes #

**How did you address these issues with this PR? What methods did you use?**

### CODE REVIEW

**Comment by Wikunia** [comment]
Make sure to run:
```julia
using JuliaFormatter
format(".")
```
in your root directory.

File: src/latexsvgfile.jl [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#file with the LaTeXSVG constant
+export LaTeXSVG
```
Comment (Wikunia) [inline src/latexsvgfile.jl:2]: Is there a reason you want to export this?

Comment (ArbitRandomUser) [reply src/latexsvgfile.jl:2]: i thought variables had to be exported ; i guess thats only for modules ?
does include("filename") work like #include in  c?

Comment (ArbitRandomUser) [reply src/latexsvgfile.jl:2]: docstring for include says , `Evaluate the contents of the input source file in the global scope of the containing module. Every module (except those defined with baremodule) has its own definition of include, which
  evaluates the file in that module.`
so export isnt needed i believe . Should i remove export ?

Comment (Wikunia) [reply src/latexsvgfile.jl:2]: export would mean that users of Javis can access it without using `Javis.` in front of it. `include` should be working fine without `export`. I would therefore remove it.

**Comment by Wikunia** [comment]
Thanks @ArbitRandomUser

**Pull request merged**